setup.py

```diff
@@ -1,45 +1,92 @@
 from __future__ import annotations
 
-import sys
+from pathlib import Path
 
-from setuptools import find_packages, setup
+import setuptools
 
-requirements = ["dials-data", "Jinja2", "procrunner"]
+from build import build
 
-setup_requirements = []
-needs_pytest = {"pytest", "test", "ptr"}.intersection(sys.argv)
-if needs_pytest:
-    setup_requirements.append("pytest-runner")
+__version_tag__ = "3.9.dev"
 
-test_requirements = ["mock", "pytest"]
-
-setup(
-    author="Diamond Light Source",
-    author_email="scientificsoftware@diamond.ac.uk",
-    classifiers=[
-        "Development Status :: 1 - Planning",
+setup_kwargs = {
+    "name": "dials",
+    "version": __version_tag__,
+    "long_description": Path(__file__).parent.joinpath("README.md").read_text(),
+    "description": "Diffraction Integration for Advanced Light Sources",
+    "author": "Diamond Light Source",
+    "license": "BSD-3-Clause",
+    "author_email": "dials-support@lists.sourceforge.net",
+    "project_urls": {
+        "homepage": "https://dials.github.io",
+        "repository": "https://github.com/dials/dials",
+    },
```

Comment:
I would suggest moving the bulk of the static information into a `setup.cfg` - happy to provide this as commit

Reply:
I thought I recalled us doing this for dxtbx, but it appears not to be the case? Did something not work?